libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -76,8 +76,9 @@ export const getFederatedContentRoot = () => {
     ? origin
     : 'https://www.adobe.com';
 
-  if (origin.includes('localhost') || origin.includes('.hlx.')) {
-    federatedContentRoot = `https://main--federal--adobecom.hlx.${origin.includes('hlx.live') ? 'live' : 'page'}`;
```

Comment:
Just to double-check: hlx.live would use `'https://www.adobe.com'` and there isn't a need for any `federal` reference, because that is already present in the path, right? One caveat to note is that federal hlx.live pages may not get the latest published content, an author would have to wait for a cache refresh from Akamai, we may want to communicate that

Reply:
Added hlx.live back for hlx.live, so just localhost/hlx.page see any changes now.
Stage resources shouldn't be cached, so that should be fine